lenovo/yoga/7/14ARH7/amdgpu/default.nix

```diff
@@ -1,24 +1,4 @@
-# Including this file will enable the AMD-GPU driver
-
-{ lib, pkgs, ... }:
-
-let
-  inherit (lib) mkDefault;
-
-in {
-  imports = [
-    ../shared.nix
-    ../../../../../common/gpu/24.05-compat.nix
-  ];
-
-  # AMD RX680
-  services.xserver.videoDrivers = mkDefault [ "amdgpu" ];
-
-  hardware = {
-    amdgpu.loadInInitrd = true;
```

Comment:
The Lenovo Yoga 7 (14ARH7) has a Ryzen CPU, and integrated AMD GPU.

IIRC, I had this enabled because of an issue with needing the AMD GPU enabled earlier in order for the framebuffer console to work correctly before Wayland or X started.

I haven't tried it disabling it since I first enabled it, but assume it's probably still beneficial to keep this on GA402X.

Reply:
Note:  Disabling VPDAU should be possible without also disabling AMD GPU?

Reply:
Apologies, and never mind this comment.
I hadn't noticed the new `../shared.nix` file.

> The Lenovo Yoga 7 (14ARH7) has a Ryzen CPU, and integrated AMD GPU.
> 
> IIRC, I had this enabled because of an issue with needing the AMD GPU enabled earlier in order for the framebuffer console to work correctly before Wayland or X started.
> 
> I haven't tried it disabling it since I first enabled it, but assume it's probably still beneficial to keep this on Yoga 7.